packages/cli/src/models/files/ZosNetworkFile.ts

```diff
@@ -0,0 +1,418 @@
+import _ from 'lodash';
+import { Logger, FileSystem as fs, bytecodeDigest, bodyCode, constructorCode, semanticVersionToString } from 'zos-lib';
+import { fromContractFullName, toContractFullName } from '../../utils/naming';
+import { ZOS_VERSION, checkVersion } from './ZosVersion';
+import ZosPackageFile from './ZosPackageFile.js';
+import { ContractWrapper } from 'zos-lib';
+
+const log = new Logger('ZosNetworkFile');
+
+interface ContractInterface {
+  address: string;
+  constructorCode: string;
+  localBytecodeHash: string;
+  deployedBytecodeHash: string;
+  bodyBytecodeHash?: string;
+  types?: any;
+  storage?: any;
+  warnings?: any;
+}
+
+interface SolidityLibInterface {
+  address: string;
+  constructorCode: string;
+  bodyBytecodeHash: string;
+  localBytecodeHash: string;
+  deployedBytecodeHash: string;
+}
+
+export interface ProxyInterface {
+  alias?: string;
+  package?: string;
+  contract?: any;
+  address?: string;
+  version?: string;
+  implementation?: string;
+}
+
+export interface DependencyInterface {
+  name?: string;
+  package?: string;
+  version?: string;
+  customDeploy?: boolean;
+}
+
+interface PackageInterface {
+  address?: string;
+}
+
+interface ProviderInterface {
+  address?: string;
+}
```

Comment:
is it ok declaring all these interfaces here? shouldn't they be somewhere else?

Reply:
We are not worrying about this sort of stuff (design architecture decisions) in this phase of the port. The idea is that we port all the primitive types, introduce simple interfaces, and at a latter stage, we decide what to do with them.